pkg/network/ebpf/c/protocols/http2/decoding-tls.h

```diff
@@ -437,8 +437,8 @@ static __always_inline void tls_fix_header_frame(tls_dispatcher_arguments_t *inf
 }
 
 static __always_inline bool tls_get_first_frame(tls_dispatcher_arguments_t *info, frame_header_remainder_t *frame_state, http2_frame_t *current_frame, http2_telemetry_t *http2_tel) {
-    // No state, try reading a frame.
-    if (frame_state == NULL) {
+    // Attempting to read the initial frame in the packet, or handling a state where there is no remainder and finishing reading the current frame.
+    if (frame_state == NULL || (frame_state->remainder == 0 && frame_state->header_length == 0)) {
```

Comment:
do we still need that change?
didn't we fix the case of having empty remainder in this PR?

Reply:
still relevant